### DISCUSSION TITLE
[FEATURE-REQUEST] tener la opción de desactivar el canal ayuda de una empresa

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#112] [3. Ideas & Feature Requests] [answered]
Hola
Sería bueno tener la opción, en el momento de crear una compañía, de activar/desactivar el canal de ayuda de usuarios. El fundamento puede ser que actualmente, el medio de contacto para consultas de clientes es mediante por el correo de soporte. Para compañías que requieran un canal de ayuda personalizado se podría activar esa opción.

Gracias

![image](https://user-images.githubusercontent.com/61419471/116273044-6b563980-a74f-11eb-978d-48e10d50214b.png)

**Comment by acedward** [comment, marked as answer]
Esto está en el roadmap! Pronto no se activarán más.

**Reply by NicolasPLG** [reply]
gracias @acedward
